docs/application-development-guide.md

```diff
@@ -3224,6 +3648,11 @@ await handle.signal(update, 300);
 Every month, a customer will be charged an amount specified by the update handler.
 The update handler is a function that takes a number and returns a numer. That number is used to update the amount the customer is charge.
 
+</TabItem>
+<TabItem value="python">
+
+Content is not available
```

Comment:
I can provide this content (I know I left it out of my README as well, though can be fairly obvious when browsing the API)